README.md

```diff
@@ -2,43 +2,43 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-
 ### Install dependencies
 
     npm install
 
 ### Setup environment file
+
 create a file at root called `.env` and add this:
 
-    REACT_APP_API=<server url>
+    REACT_APP_API_URL=<server url>
 
 ### Run App
+
     npm start
 
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 The page will reload if you make edits.\
 You will also see any lint errors in the console.
 
-
 ### Check Lint errors
+
     npm run lint
 
 ### Fix Lint errors
+
     npm run lint:fix
 
 <br>
 
 ### Members
 
 (example)
-|Photo | Name  | Mail | Github
+|Photo | Name | Mail | Github
 | :-----: | :-----: | :-----: | :-----: |
 <img src="https://avatars.githubusercontent.com/u/20587232?v=4" height="50" width="50">| Esteban Frare | esteban.frare@radiumrocket.com | [@estefrare](https://github.com/estefrare)
 
-
-
 <br>
 
 ### License & Copyright
 
-© Radium Rocket "Become a Software Professional 2022"
\ No newline at end of file
+© Radium Rocket "Become a Software Professional 2022"
```

Comment:
Remove package-lock.json from your changes

Reply:
there is no changes in package-lock.json, I had just commented something in an attempt to fix a bug, but I already keep it as it was originally.